src/main/java/com/sforce/cd/apexUnit/client/connection/SFDCSessionRenewer.java

```diff
@@ -0,0 +1,19 @@
+package com.sforce.cd.apexUnit.client.connection;
```

Comment:
license header missing for this new file. Please follow the short form header as available in https://github.com/forcedotcom/ApexUnit/commit/83a83e2c1165b56cc3e5b3f07c9e18cdf0194a11

Can you also update the license header for all files with the new short form in a different pull request? or may be open as a